templates/openwisp2/settings.py

```diff
@@ -53,6 +53,11 @@
 {% if openwisp2_firmware_upgrader %}
     'openwisp_firmware_upgrader',
     'private_storage',
+{% endif %}
+{% if openwisp2_monitoring %}
+    'openwisp_monitoring.monitoring',
+    'openwisp_monitoring.device',
+    'openwisp_monitoring.check',
```

Comment:
I don't think this will make any much difference, but let's move this block up after `openwisp_controller.connection`. This will make the order consistent with [README of openwisp-monitoring](https://github.com/openwisp/openwisp-monitoring#setup-integrate-in-an-existing-django-project).